test/byContract/registries/registries.test.ts

```diff
@@ -534,4 +565,10 @@ describe('Registries', () => {
 
     const computeNameHash = (address: string) =>
         ethers.utils.namehash(`${address.substring(2)}.addr.reverse`);
+
+    function BN2Address(n: BigNumber): string {
```

Comment:
To understand, don't want to use the one from `tools/` ?

Reply:
Oups, I thought it would not work but it is ! thanks, look [remove created BN2Address and import tools instead](https://github.com/iExecBlockchainComputing/PoCo/pull/124/commits/1851d8ed557785efab666e4b3f124307b6f6ea54)